experimental/material_3_demo/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: material_3_demo
+description:
+  A Flutter project showcasing supported Material 3 components, typography, color system and elevation.
+  Supports different light/dark mode, color seed, and comparison to Material 2.
+
+publish_to: "none"
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.17.0-0 <3.0.0"
```

Comment:
Any particular reason for `2.17.0-0`? I figure this should really be `2.18.0-0`.

Reply:
Don't have any particular reasons for that version. It's just auto-generated. Thanks for noticing this. Changed!